src/excmds.ts

```diff
@@ -686,6 +686,21 @@ export async function tabmove(index = "0") {
     let newindex: number
     if (index.startsWith("+") || index.startsWith("-")) {
         newindex = Math.max(0, Number(index) + aTab.index)
```

Comment:
I didn't actually benchmark any of this and I'd be pleased if anyone has input on how to do so efficiently, but I embed the work of constructing Number(index) here before checking if the newindex doesn't do anything (newindex == aTab.index which is equivalent to Number(index) == 0) just to avoid calling the Number constructor twice if we were to check Number(index) == 0 separately or setting newindex twice. I'd say someone who actually sends +0 or -0 to tabmove could live with this extra bit of computation before the short circuit.

Reply:
Don't need to clamp this value. Array.slice does its own clamping.